src/bundle/Resources/public/scss/_mixins.scss

```diff
@@ -1,34 +1,37 @@
 @mixin checkbox-icon() {
     border: 2px solid $ez-color-secondary;
-    padding: .2rem;
+    padding: 4px;
     margin: 0;
-    line-height: 1;
-    border-radius: 5px;
+    line-height: .5;
+    border-radius: .25rem;
     position: relative;
     z-index: 1;
     cursor: pointer;
 
     &:before {
         content: "";
         position: absolute;
-        width: 2.3rem;
-        height: 2.3rem;
+        width: 2rem;
+        height: 2rem;
         top: 50%;
         left: 0;
         background-color: $ez-color-secondary;
         z-index: -1;
-        border-radius: 5px;
+        border-radius: .25rem;
         transform: translate(calc(100% + .3rem), -50%);
-        transition: all .2s $ez-admin-transition;
+        transition: all .2s ease-in;
```

Comment:
Just curiosity, what was wrong with the previous timing function?

Reply:
There’s nothing wrong about the cubic bezier option :-)

I am more inclined to options like ease-in because they depict a natural movement that is aligned with user expectations when an event is triggered; and also a movement that doesn’t demand breaking user’s attention.

Ease-in with a long timing undermines usability. Here is proposed .2s, but at the same time the effect is not that fast that breaks user’s attention.